inventory/host_vars/donalo.local/config.yml

```diff
@@ -8,3 +8,6 @@ sys_admins:
   - name: "{{ development_user }}"
     ssh_key: "~/.ssh/id_rsa.pub"
     state: present
+
+database_metabase_user: metabase
+database_metabase_password: metabase
```

Comment:
I'd rather use `metabase_database_*` so it's more like a namespace but it's just a personal preference.

Reply:
It's actually the `metabase_user` for the `database`, so the namespace is correct IMO.